components/selection/selection-summary.js

```diff
@@ -50,10 +50,13 @@ class Summary extends LocalizeCoreElement(SelectionObserverMixin(LitElement)) {
 	}
 
 	_updateSelectSummary() {
-		if (this._provider && this._provider.selectionSingle) return;
+		if (this._provider?.selectionSingle) {
+			this._summary = null;
+			return;
+		}
 
 		let count;
-		if (this._provider && this._provider.selectionCountOverride !== undefined) {
+		if (this._provider?.selectionCountOverride !== undefined) {
```

Comment:
Is the `_provider` check necessary here? Given the change above, it looks like we should never reach this code if  `_provider` is `null`/`undefined`.

Reply:
Maybe I am missing something. It looks like if the `_provider` isn't set, we _will_ wipe out the summary and return above.

Reply:
Sorry, I was reading it as "provider is undefined or selection single". Is that what you meant it to be @eKoopmans ?

Reply:
Oops, I mostly thought this was cleaning things up, but you're right that this isn't the same - e.g. if `this._provider` is `null`, this check will now pass. I'll revert these optional chaining bits.

@dbatiste I ended up changing the logic from what we discussed yesterday. Yesterday we had decided that `_summary` should always be `null` if there's no provider, but that was breaking for scenarios that are pageable but not selectable.

Reply:
Cool... yeah... easy to get tripped up with optional-chaining on statements like this one. 😄